### ISSUE TITLE
Crashing because of `json.datapoints` is undefined

### ISSUE THREAD

**Issue by iRonin** [gchokov/homebridge-foobot#3]
I requested quota upgrade and received new key and I got this when I added the key:

```
TypeError: Cannot read property 'length' of undefined
    at Foobot.<anonymous> (/homebridge/node_modules/homebridge-foobot/index.js:313:37)
    at Foobot.<anonymous> (/homebridge/node_modules/homebridge-foobot/index.js:191:6)
```

When I query the API via https://api.foobot.io/apidoc/index.html it works correctly.

**Comment by iRonin** [comment]
For some reason the json is:

```
{ message: 'invalid key provided' }
```

**Comment by gchokov** [comment]
Fixed in v0.5.0 with your contribution. Thanks!
